docker-compose.yml

```
@@ -0,0 +1,48 @@
+services:
+  redis:
+    image: redis:7-alpine
+    platform: linux/amd64
+    volumes:
+      - redis-data:/data
+    networks:
+      - across_relayer_network
+    restart: always
+    command: redis-server
+    healthcheck:
+      test: redis-cli ping
+      interval: 15s
+      timeout: 5s
+      retries: 3
+
+  relayer_1:
+    image: ${AWS_ECR_REGISTRY}/${AWS_ECR_REPOSITORY}:${ACROSS_RELAYER_IMAGE_TAG}
+    platform: linux/amd64
+    depends_on:
+      redis:
+        condition: service_healthy
+    volumes:
+      - ./scripts/lisk/docker/${NETWORK:-sepolia}/containerStartBot_1.sh:/home/lisk/across-relayer/scripts/start.sh
```

```suggestion
      - ./scripts/lisk/docker/${NETWORK:-mainnet}/containerStartBot_1.sh:/home/lisk/across-relayer/scripts/start.sh
```
Should it be `mainnet` (assuming this PR goes onto the production server) ?

---

I'll update it to `mainnet`. But leaving it `sepolia` also shouldn't be an issue as the `NETWORK` environment variable is set from the appropriate AWS secrets within L19 of `scripts/lisk/docker/setEnvVariables.sh`.